DependencyInjection/SmartboxIntegrationFrameworkExtension.php

```diff
@@ -308,7 +308,15 @@ protected function loadHandlers(ContainerBuilder $container)
             $handlerDef->addMethodCall('setThrottleDelay', [$handlerConfig['throttle_delay']]);
             $handlerDef->addMethodCall('setThrottleStrategy', [$handlerConfig['throttle_strategy']]);
             $handlerDef->addMethodCall('setThrottleDelayFactor', [$handlerConfig['throttle_delay_factor']]);
-            $handlerDef->addMethodCall('setThrottleURI', [$handlerConfig['throttle_uri']]);
+
+            // If there is no throttle_uri set, we need to set it with something, so we will use the retry uri.
+            // This will maintains the current behaviour, but allows us to explicitly set a throttle_uri
+            if(isset($handlerConfig['throttle_uri'])){
+                $handlerDef->addMethodCall('setThrottleURI', [$handlerConfig['throttle_uri']]);
+            }
+            else{
+                $handlerDef->addMethodCall('setThrottleURI', [$handlerConfig['retry_uri']]);
```

Comment:
You don't need to write 2 times the same method call. For instance
```
$throttleUrl = isset($handlerConfig['throttle_uri'])) ? $handlerConfig['throttle_uri'] : $handlerConfig['retry_uri']

$handlerDef->addMethodCall('setThrottleURI', [$throttleUrl]);
```

Reply:
yep, that looks much cleaner, updated